### ISSUE TITLE
Steps to configure Ethereum with Peatio

### ISSUE THREAD

**Issue by sharmaabhishek2207** [openware/peatio#591]
Hi All,

Can anyone please help me to get the steps to configure Ethereum with Peatio? In setup documentation for ubuntu, only bitcoin steps are provided. Thanks in advance.

Regards
Abhishek

**Comment by oskanddr** [comment]
Hi @sharmaabhishek2207,

I have ETH implemented in private Peatio fork. I can help you to deploy it or add to existing Peatio installation. 

Contact me: nimpo1074@gmail.com.

Regards,
Oskar

**Comment by yivo** [comment]
Hi @sharmaabhishek2207,

Right now Peatio doesn't support ETH but we will ship it very soon. Configuration will be available through currencies.yml.

**Comment by riddlez666** [comment]
when is soon?

**Comment by sg552** [comment]
I studied source code and found the config file --  config/seed/currencies.yml  

However I don't know which ethereum client to use, geth or parity .  

According to this [link](https://github.com/peatio/peatio/issues/668) , someone said that parity is better than geth for Peatio.  

and this link: https://github.com/InfraexDev/peatio/blob/stable/doc/eth.md  ,  is another fork of Peatio. showing how to setup ethereum staffs. 

well, I will give my answer after tried.

**Comment by sharmaabhishek2207** [comment]
Any luck on ethereum setup?

**Comment by sg552** [comment]
well, I stopped my work on Ethereum. 

I installed "Geth", and made it running, Peatio could generate new account via PRC-JSON call.  

however, once I transferred little ETH to the account that Peatio holds, Peatio could not obtain the incoming.  That is said, Peatio's ETH account balance kept showing "0". ( 0.1 ETH expected) 

so the "Transfer" function is not test . 

I will try this function once I finished my job on hands.
